README.rst

```diff
@@ -0,0 +1,166 @@
+courseraresearchexports
+=======================
+
+.. image:: https://travis-ci.org/coursera/courseraresearchexports.svg
+    :target: https://travis-ci.org/coursera/courseraresearchexports
+
+This project is a library consisting of a command line interface and a client
+for interacting with Coursera's research exports.
+
+Installation
+------------
+
+To install this sdk, simply execute::
+
+    sudo pip install courseraresearchexports
+
+`pip <https://pip.pypa.io/en/latest/index.html>`_ is a python package manager.
+If you do not have ``pip`` installed on your machine, please follow the
+installation instructions for your platform found at:
+https://pip.pypa.io/en/latest/installing.html#install-or-upgrade-pip
+
+Note: the tool requires ``docker`` to already be installed on your machine.
```

Comment:
using the docker part of the tool requires docker, simply generating/managing download links for exports does not.